app/src/main/resources/application.yml

```diff
@@ -159,8 +355,10 @@ nakadi:
   stream:
     maxStreamMemoryBytes: 10_000 # ~10 Kb
     max.commitTimeout: 5 # seconds
+    timeoutMs: 31536000000 # 1 year
```

Comment:
is this from upstream or something we added?

Reply:
Yes. It is from upstream